<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjectDialog</class>
 <widget class="QDialog" name="AddObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="tableLabel">
     <property name="text">
      <string>Select Table:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="tableComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="recordLabel">
     <property name="text">
      <string>Enter Record String:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="recordEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddObjectDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
